### DISCUSSION TITLE
[3.50] Support mix feature in global animation and private animation both

### DISCUSSION THREAD

**Discussion by rexrainbow** [photonstorm/phaser#5916] [Feature Requests]
In 3.50-beta4, mix is set in animation manager [if 2 global animations are existed](https://github.com/photonstorm/phaser/blob/master/src/animations/AnimationManager.js#L173). It can't support mix for private animation created by [`sprite.anims.create()`](https://github.com/photonstorm/phaser/blob/master/src/animations/AnimationState.js#L1694). It is nice that mix feature also can support global animation and private animation both.
A possible solution might be, put responding mix in each [animation object](https://github.com/photonstorm/phaser/blob/master/src/animations/Animation.js), to store transition delay of previous animation.

**Comment by photonstorm** [comment]
Yeah I did consider this. I was thinking about making mix based on a frame name/number, instead of delay, too.
